cf_remote/commands.py

```diff
@@ -476,10 +476,25 @@ def uninstall(hosts):
     return errors
 
 def deploy(hubs, directory):
-    assert(directory.endswith("/masterfiles"))
-    assert(os.path.isfile(directory + "/autogen.sh"))
+    directory = os.path.abspath(os.path.expanduser(directory))
+    log.debug(f"Deploy directory expanded to: {directory}")
+    if directory.endswith("/"):
+        directory = directory[0:-1]
+
+    if not directory.endswith("/masterfiles"):
+        log.error("The masterfiles directory to deploy must be called 'masterfiles'")
+        return 1
+    if not os.path.isdir(directory):
+        log.error(f"'{directory}' must be a directory")
+        return 1
+    if not os.path.isfile(directory + "/autogen.sh"):
```

Comment:
Could use `os.path.join()`, but I prefer the simplicity of just `+ "/..."`.

Reply:
yep